docs/sources/troubleshooting/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Solve common issues with Explore Logs.
+keywords:
+  - Logs
+  - Explore
+  - Labels
+  - Analysis
+menuTitle: Troubleshooting
+title: Troubleshooting Explore Logs
+weight: 700
+---
+
+# Troubleshooting
+
+{{% admonition type="caution" %}}
+Explore Logs is currently in [public preview](/docs/release-life-cycle/). Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available.
+Please [send us any feedback](https://forms.gle/1sYWCTPvD72T1dPH9).
+{{% /admonition %}}
+
+This page address common issues when getting started and using Explore Logs.
+
+## There are no services
```

Comment:
There could be also no services if the Volume API is not activated 

see https://github.com/grafana/loki/pull/12628 it's default on in 3.1 or helm but not in 3.0

Reply:
thx